### DISCUSSION TITLE
Toggle ID field as Presentable

### DISCUSSION THREAD

**Discussion by yanc0** [pocketbase/pocketbase#3537] [Q&A] [answered]
Hi folks :wave:,

First of all, thanks for this amazing work on Pocketbase, it's really fun and powerful to use.

I have meaningful IDs on one collection and I want make them "Presentable" when a Record relates to those. On this collection, there is an `email` field, besides the fact that it is not toggled, it is used automatically in the relation listings. What can I do to force Pocketbase to use `id` as presentable field ?

Thanks !

**Comment by ganigeorgiev** [comment, marked as answer]
> What can I do to force Pocketbase to use id as presentable field ?

It is not possible at the moment. There are plans in the future to allow control over the default system fields (`id`, `email`, `username`, etc.) and treat them as any other regular field but for now this is left aside.

**Reply by yanc0** [reply]
Thanks @ganigeorgiev for your (superfast) answer.
